aeroprom.conf.dev

```diff
@@ -28,17 +29,28 @@ db_port=3000
 
 # certificate file
 cert_file=""
+
 # key file
 key_file=""
+
+# Passphrase for encrypted key_file. Supports below formats,
+# 1. Passphrase directly                 - "<passphrase>"
+# 2. Passphrase via file                 - "file:<file-that-contains-passphrase>"
+# 3. Passphrase via environment variable - "env:<environment-variable-that-holds-passphrase>"
+key_file_passphrase="${AS_KEY_FILE_PASSPHRASE}"
+
 # node TLS name for authentication
 node_tls_name=""
+
 # root certificate file
 root_ca=""
 
 # authentication mode: internal (for server), external (LDAP, etc.)
 auth_mode=""
+
 # database user
 user=""
+
 # database password
 password=""
```

Comment:
We may want to remove these two arguments as well. It was a mistake on my part that I didn't remove them before release.

Reply:
Resolving as discussed.